modules/nf-core/mitohifi/findmitoreference/main.nf

```diff
@@ -0,0 +1,47 @@
+process MITOHIFI_FINDMITOREFERENCE {
+    tag '$species'
+    label 'process_low'
+
+    // Docker image available at the biocontainers Dockerhub
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://biocontainers/mitohifi:3.0.0_cv1':
+        'docker.io/biocontainers/mitohifi:3.0.0_cv1' }"
```

Comment:
What's happening there?

Reply:
The module is outdated. I fixed it up: https://github.com/nf-core/modules/blob/master/modules/nf-core/mitohifi/findmitoreference/main.nf, just not in this code base yet.